wallet/wallet-cli-commands/src/lib.rs

```diff
@@ -960,12 +986,19 @@ pub enum WalletCommand {
         /// The transaction outputs, in the format `transfer(address,amount)`
         /// e.g. transfer(tmt1q8lhgxhycm8e6yk9zpnetdwtn03h73z70c3ha4l7,0.9)
         outputs: Vec<String>,
-        /// You can choose what utxos to spend (space separated as additional arguments). A utxo can be from a transaction output or a block reward output:
+        /// You can choose what utxos to spend (space separated as additional arguments).
+        ///
+        /// A utxo can be from a transaction output or a block reward output:
         /// e.g tx(000000000000000000059fa50103b9683e51e5aba83b8a34c9b98ce67d66136c,1) or
         /// block(000000000000000000059fa50103b9683e51e5aba83b8a34c9b98ce67d66136c,2)
         #[arg(long = "utxos", default_values_t = Vec::<String>::new())]
         utxos: Vec<String>,
 
+        /// This specifies that instead of a (hex encoded) PartiallySignedTransaction
+        /// the result should be a (hex encoded) "simple" transaction.
+        ///
+        /// Note that both variants are accepted by account-sign-raw-transaction,
```

Comment:
unless signing with a cold wallet which will need the partially signed tx format as it cannot fetch the utxos.